index.html

```diff
@@ -492,6 +492,20 @@ <h3 id="PartnerAPIs-Venues">Venues</h3>
                   </ul>
               </td>
           </tr>
+          <tr>
+              <td>GET</td>
+              <td>/partner/v1/venues/byVendorVenueId/&lt;vendorVenueId&gt;</td>
+              <td><a href="#AtomVenueForVendorVenueExample">Get Atom Venue for VendorVenueId</a></td>
```

Comment:
To match the current descriptions:
"Query for venues by Vendor Id"

Reply:
i want to be specific about atom venue vs vendor venue in terms of ids